core/src/main/scripts/importer/validateData.py

```diff
@@ -1828,6 +1835,23 @@ def checkValidationColumns(self, data):
 
         return True
 
+    def checkOffPanelVariant(self, data, normalized_gene, panel_id, hugo_symbol, entrez_id):
+        try:
+            normalized_gene = int(normalized_gene)
+        except:
+            pass
```

Comment:
Should we throw a warning or an error when we cannot parse a `normalized_gene`?

Reply:
The normalized_gene is either a entrez id or None(when mapping to gene table is not possible). If its None a warning is thrown here https://github.com/cBioPortal/cbioportal/blob/3744576fd8179368f7896703729c3b408e0b0cda/core/src/main/scripts/importer/validateData.py#L725

Reply:
Thanks!